web/app/controllers/drivingSchoolCtrl.js

```diff
@@ -48,10 +48,60 @@ exports.getSchool = function(schoolId, callback, error) {
 };
 
 // TODO: Add update method to add a student to be added to the school
+// WHAT DOES THIS EVEN MEAN?!? ^^^^^
+// I read this like 5 times and can't figure it out
```

Comment:
Basically to update a school's student array with the student to be added.  @DylanWalseth